README.md

```diff
@@ -614,6 +621,13 @@ Our plan for the nearest future:
 We have much more ideas to work on.
 See more detailed plan in the dedicated [GitHub Project page](https://github.com/kowainik/stan/projects/1).
 
+## Users
+
+Stan is known to be used in the following companies:
```

Comment:
Alternative: adopted by, integrated to the companies' processes etc.?

Reply:
I like "Adopted by" 👍🏻 Does the following look okay?

```suggestion
Stan is known to be adopted by the following companies:
```